routes/rooms.go

```diff
@@ -56,6 +74,77 @@ func GetEmergencyRoomById(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, resp)
 }
 
+// GetEmergencyRooms godoc
+// @Summary    get all emergency rooms
+// @Tags       emergency-rooms
+// @Produce    json
+// @Param      page                            query       uint                    false   "0-indexed page number, 0 is assumed when omitted"
+// @Success    200                             {object}    GetMultipleERsResponse
+// @Failure    501                             {object}    HTTPErrorResponse
+// @Router     /er                             [get]
+func GetEmergencyRooms(ctx *gin.Context) {
+	log, logCtx := logging.GetRequestLogger(ctx)
+	db := models.GetDB(logCtx)
+
+	erPageRaw := ctx.Param("page")
+	log.Debug().Str("getting all emergency rooms on page", erPageRaw).Send()
```

Comment:
`.Str(key, value)` attaches an additional field `key` to the log output. You maybe wanted to do sth like this;

```suggestion
	log.Debug().Str("page", erPageRaw).Msg("getting all emergency rooms on page")
```

or 

```suggestion
	log.Debug().Msgf("getting all emergency rooms on page %s", erPageRaw)
```

or maybe just

```suggestion
	log.Debug().Str("page", erPageRaw).Send()
```